pkg/bsp-imx/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+FROM lfedge/eve-alpine:8e3ddd8016f778932e9c204a3ec195ed74d1600a as build-base
+RUN mkdir /bsp
+
+FROM build-base AS build-amd64
+FROM build-base AS build-riscv64
+FROM build-base AS build-arm64
+ENV BUILD_PKGS bash binutils-dev build-base bc bison flex openssl-dev util-linux-dev swig gnutls-dev perl python3 python3-dev py3-setuptools py3-pycryptodome py3-elftools
+RUN eve-alpine-deploy.sh
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+
+ENV UBOOT_TARGETS "imx8mq_evk"
+ENV ATF_TARGETS "imx8mq"
+
+RUN mkdir /uboot-firmwares
+
+# ATF
+WORKDIR /imx-atf
+# last commit from imx_5.4.70_2.3.0 branch
+ENV ATF_COMMIT 15e8ff164a8becfddb76cba2c68eeeae684cb398
+ADD https://github.com/nxp-imx/imx-atf.git#${ATF_COMMIT} .
+RUN for target in ${ATF_TARGETS}; do \
+        make clean && make -j "$(getconf _NPROCESSORS_ONLN)" PLAT=$target bl31 ;\
+        mv build/imx8mq/release/bl31.bin /uboot-firmwares/${target}-bl31.bin ;\
```

Comment:
imx8mq is hard coded here, it should be ${target}

Reply:
Fixed, thanks!